src/main/java/edu/ntnu/fullstack/prosjekt/quizzer/domain/dto/QuizDto.java

```diff
@@ -1,4 +1,37 @@
 package edu.ntnu.fullstack.prosjekt.quizzer.domain.dto;
 
+import edu.ntnu.fullstack.prosjekt.quizzer.domain.entities.UserEntity;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * The QuizDto class is a mirror of the QuizEntity class, with the intention of creating
+ *  * a separation between user input/output and database objects.
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
 public class QuizDto {
```

Comment:
Vi burde vel kanskje ha med questions i DTO-en, med tanke på at frontend kommer til å trenge de når man åpner en quiz

Reply:
Bruker denne DTO-en for å hente quizzes uten inkluderte spørsmål. Lager en annen DTO med innholdet til den spesifikke quizen.